pages/country.js

```diff
@@ -27,17 +27,20 @@ export default class Country extends React.Component {
     let results = await Promise.all([
       // XXX cc @darkk we should ideally have better dedicated daily dumps for this view
       client.get('/api/_/test_coverage', {params: {'probe_cc': countryCode}}),
-      client.get('/api/_/country_overview', { params: {'probe_cc': countryCode}})
+      client.get('/api/_/country_overview', { params: {'probe_cc': countryCode}}),
+      client.get(`https://ooni.io/tags/country-${countryCode.toLowerCase()}/index.json`)
```

Comment:
What happens if there is not tag for the selected country? I think we should explicitly handle that case.